lib/mintel/alerts/mintel-workloads.libsonnet

```diff
@@ -43,6 +43,19 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'KubeCPUUsageOutsideNormalParameters',
```

Comment:
we should not apply this so widely or it will fire all the time, i think we should select which ones we want to alert on starting with flux